app/src/controllers/shop/services/ReservationService.ts

```diff
@@ -38,6 +46,23 @@ const ReservationService: ReservationServiceInterface = {
     return ReservationRepository.fetchShopReservations(shopId, limit)
   },
 
+  async getTotalsalesForShop(shopId) {
+    const completedReservations = await
+    ReservationRepository.fetchCompletedShopReservationsWithStyilstPriceAndMenuPrice(
+      shopId,
+    )
+
+    const allSales = completedReservations.map(cr => {
+      if (!cr.stylistPrice) {
+        Logger.debug('No stylist Price')
+        throw new NotFoundError()
+      }
+      return pricePerSession(cr.stylistPrice, cr.menuPrice)
+    })
+
+    const totalSales = getTotalSales(allSales)
```

Comment:
if we're not reusing this method, i think we can just do the logic here instead of making a method for it?

Reply:
only made it to make it cleaner.. but now i think about it i guess it would be easier to read if we write there